storage/innobase/xtrabackup/src/xbcloud/xbcloud-t.cc

```diff
@@ -558,15 +558,15 @@ TEST(azure_signer, basicDNS) {
   req.append_payload("test", 4);
   Azure_signer signer(
       "my-storage-account",
-      "zUfvsKXc6+2RMJCwvnElnG/"
-      "Kk7wxQ8V4TPXIuZ53qFwJNtpLUEjYdBe9iGTkMgwUGFHVfFgn2qkgoqDP/b3OAg==",
+      "zUfvsKXcAO2RMJCwvnElnG/"
```

Comment:
Now this key contains a NULL byte when decoded, and it ends in `\r` to check that the request signing is correctly done